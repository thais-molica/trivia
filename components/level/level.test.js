import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Level from "./index";

it("Level renders correctly", () => {
  const tree = renderer.create(<Level id="easy"/>).toJSON();
  expect(tree).toMatchSnapshot();
});
