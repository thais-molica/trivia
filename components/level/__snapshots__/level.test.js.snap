// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level renders correctly 1`] = `
<div
  className="jsx-3320805743"
>
  <img
    alt="level"
    className="jsx-1677436475"
    src={[Function]}
  />
  <img
    alt="level"
    className="jsx-1710938339"
    src={[Function]}
  />
  <img
    alt="level"
    className="jsx-1710938339"
    src={[Function]}
  />
  <span
    className="jsx-3320805743"
  >
    easy
  </span>
</div>
`;
